### PR TITLE
Added ISSUE-TEMPLATE.md

### PR DESCRIPTION
**Pull request by DaanVink**
This allows for more standardized issue reports and easier reading of said reports

### CODE REVIEW

File: ISSUE-TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+**Category:**
```
Comment (DFoxinator) [inline ISSUE-TEMPLATE.md]: I'm thinking maybe it would be good to provide some options for this one?

Comment (DaanVink) [reply ISSUE-TEMPLATE.md]: Indeed it would be
I suggest **Category: (API, Account, Web, App)**

**Comment by chhanganivarun** [comment]
Thanks @DaanVink

File: ISSUE-TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+**Category:** 
+
+**Description of bug:**
```
Comment (chhanganivarun) [inline ISSUE-TEMPLATE.md]:
```
**Description of bug/feature:**
```

**Comment by Skayo** [comment]
It seems like the fork isn't existing anymore. I will try to come up with a new solution...

**Comment by Skayo** [comment]
Overworked in #500

**Pull request closed without merging**